package.json

```diff
@@ -72,6 +72,7 @@
     "@babel/cli": "^7.1.2",
     "@babel/core": "^7.1.2",
     "@mdi/font": "^2.8.94",
+    "feather-icons": "^4.10.0",
```

Comment:
Was this manually inserted? Everything else in ordered by name but this and it stands out a bit, I think yarn and npm inserts everything ordered 🙂